Content.Server/Botany/Systems/BotanySystem.Seed.cs

```diff
@@ -140,18 +140,18 @@ public IEnumerable<EntityUid> Harvest(SeedData proto, EntityUid user, int yieldM
 
         var name = Loc.GetString(proto.DisplayName);
         _popupSystem.PopupCursor(Loc.GetString("botany-harvest-success-message", ("name", name)), user, PopupType.Medium);
-        return GenerateProduct(proto, Transform(user).Coordinates, yieldMod);
+        return GenerateProduct(proto, potencyBonus, Transform(user).Coordinates, yieldMod);
     }
 
-    public IEnumerable<EntityUid> GenerateProduct(SeedData proto, EntityCoordinates position, int yieldMod = 1)
+    public IEnumerable<EntityUid> GenerateProduct(SeedData proto, float potencyBonus, EntityCoordinates position, float yieldMod = 1f)
     {
         var totalYield = 0;
         if (proto.Yield > -1)
         {
-            if (yieldMod < 0)
-                totalYield = proto.Yield;
+            if (yieldMod < 0f)
+                totalYield = 0;
```

Comment:
Should this be zero?

Reply:
Seemed to me that if yieldmod ever were to reach into the negatives (which should not be possible in current behaviour, but still), the resulting yield should end up as zero. Or one.
The original code seemed more like, uh, a "panic" fallback than properly handling negative values.